### PR TITLE
First draft of breakfast recipe

### PR DESCRIPTION
**Pull request by threedom**
Everyone, feel free to add to it.

## 🎯 Objective

Made the first draft to get the recipe started.
Goal is to get to the best pancake breakfast possible.

## 🛠️ Changes made

Added basic recipe structure for pancakes and eggs.

## 👥 Who needs to review this Pull Request? 

@fernandomorenopayro @smanda1a382j @csmith42

### CODE REVIEW

**Review by fernandomorenopayro — APPROVED** [review]
It looks great! Super specific. 👍

File: breakfast/readme.md [hunk 1]
```diff
@@ -1,21 +1,62 @@
...
+
+Serving
+- Place pancakes in half moon shape on plate
+- Add eggs in the center
+- Serve with maple sirup on the side
 
 ### Notes:
 
+- Ideas on making it better?
+- Ideas on beverages?
```
Comment (smanda1a382j) [inline breakfast/readme.md:61]: Fresh squeezed orange juice

**Review by csmith42 — APPROVED** [review]
Looks great - recipe seems complete with ingredients and steps. I like the breakout of the recipe by item (pancake vs. eggs)

**Review by fernandomorenopayro — APPROVED** [review]
Perfect 👍

**Review by smanda1a382j — COMMENTED** [review]
Reviewed for completeness of ingredients and steps necessary to execute.  Looks good.

**Pull request merged**